lib/codemirror.dart

```diff
@@ -738,7 +738,7 @@ class Doc extends ProxyHolder {
    * content of the selections.
    */
   Iterable<String> getSelections([String lineSep]) =>
-      callArg('getSelections', lineSep);
+      new List<String>.from(callArg('getSelections', lineSep));
```

Comment:
I assume callArg() is not an iterable, so you're not using `.cast<String>()` here?

Reply:
nvm, I had thought it wasn't an iterable but it is.  Fixed.